server/game/cards/attachments/07/catapultonthewall.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../../drawcard.js');
+
+class CatapultOnTheWall extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Kneel Catapult and attached character',
+            condition: () => this.game.currentChallenge,
+
+            // This is not a nice interaction for users, but `ability.costs` would need to be updated with a different method to allow something like attachment.parent.kneel() (I think)
+            cost: [
+                ability.costs.kneelSelf(),
+                ability.costs.kneel(card => card === this.parent)
+            ],
+            target: {
+                activePromptTitle: 'Select character to kill',
+                cardCondition: card => this.game.currentChallenge.isAttacking(card)
```

Comment:
oh right, whoops.  Got so caught up with trying to figure out the kneel that I forgot about making sure the card was correct, apologies.